firmware/boron-ins-fsm/src/im21door.cpp

```diff
@@ -57,7 +57,7 @@ doorData checkIM21(){
   static int initialDoorDataFlag = 1;
   static doorData previousDoorData = {0x00, 0x00, 0};
   static doorData currentDoorData = {0x00, 0x00, 0};
-  static doorData returnDoorData = {0x99, 0x99, 0};
+  static doorData returnDoorData = {0x00, 0x00, 0};
```

Comment:
That is a good point, I wish I had thought of that.

Reply:
Since the door sensors send a heartbeat every 10 mins, what do you think about only reseting once it had been over 9 minutes since the last heartbeat and as long as we are not in a session? Time since the lastdoorheartbeat is already stored in doorHeartbeatReceived in the im21door.cpp so it shouldn't be too hard to implement. This would make us wrong for only about 1 min which is much better than 10.

Reply:
Great idea. Looks good.